2024-01-08 12:05:15,226 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Configure stats pid to 21055
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Loading settings from /home/jo/.config/wandb/settings
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Loading settings from /home/jo/code/kaggle/wandb/settings
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_init.py:_log_setup():524] Logging user logs to /home/jo/code/kaggle/wandb/run-20240108_120515-v5e4pg4b/logs/debug.log
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_init.py:_log_setup():525] Logging internal logs to /home/jo/code/kaggle/wandb/run-20240108_120515-v5e4pg4b/logs/debug-internal.log
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_init.py:init():564] calling init triggers
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.005, 'momentum': 0.9, 'epochs': 5, 'loss_function': 'binarycrossentropy', 'architecture': 'MLP'}
2024-01-08 12:05:15,227 INFO    MainThread:21055 [wandb_init.py:init():589] re-initializing run, found existing run on stack: 5blv7hvs
2024-01-08 12:05:15,229 INFO    MainThread:21055 [wandb_run.py:_finish():1962] finishing run jcrone/Titanic/5blv7hvs
2024-01-08 12:05:15,230 ERROR   MainThread:21055 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-01-08 12:05:15,230 INFO    MainThread:21055 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-08 12:05:15,230 INFO    MainThread:21055 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-01-08 12:05:15,230 INFO    MainThread:21055 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-08 12:05:15,230 INFO    MainThread:21055 [wandb_run.py:_restore():2185] restore
2024-01-08 12:05:15,230 INFO    MainThread:21055 [wandb_run.py:_restore():2191] restore done
2024-01-08 12:05:18,974 INFO    MainThread:21055 [wandb_run.py:_footer_history_summary_info():3837] rendering history
2024-01-08 12:05:18,974 INFO    MainThread:21055 [wandb_run.py:_footer_history_summary_info():3869] rendering summary
2024-01-08 12:05:18,978 INFO    MainThread:21055 [wandb_run.py:_footer_sync_info():3796] logging synced files
2024-01-08 12:05:19,045 INFO    MainThread:21055 [wandb_init.py:init():614] starting backend
2024-01-08 12:05:19,045 INFO    MainThread:21055 [wandb_init.py:init():618] setting up manager
2024-01-08 12:05:19,047 INFO    MainThread:21055 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-01-08 12:05:19,047 INFO    MainThread:21055 [wandb_init.py:init():624] backend started and connected
2024-01-08 12:05:19,053 INFO    MainThread:21055 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-01-08 12:05:19,053 INFO    MainThread:21055 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-01-08 12:05:19,054 INFO    MainThread:21055 [wandb_init.py:init():716] updated telemetry
2024-01-08 12:05:19,057 INFO    MainThread:21055 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-01-08 12:05:20,715 INFO    MainThread:21055 [wandb_run.py:_on_init():2254] communicating current version
2024-01-08 12:05:20,854 INFO    MainThread:21055 [wandb_run.py:_on_init():2263] got version response 
2024-01-08 12:05:20,854 INFO    MainThread:21055 [wandb_init.py:init():800] starting run threads in backend
2024-01-08 12:05:20,900 INFO    MainThread:21055 [wandb_run.py:_console_start():2233] atexit reg
2024-01-08 12:05:20,901 INFO    MainThread:21055 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-08 12:05:20,901 INFO    MainThread:21055 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-08 12:05:20,901 INFO    MainThread:21055 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-08 12:05:20,902 INFO    MainThread:21055 [wandb_init.py:init():841] run started, returning control to user process
2024-01-08 12:08:01,627 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-01-08 12:08:01,627 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Configure stats pid to 21055
2024-01-08 12:08:01,627 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Loading settings from /home/jo/.config/wandb/settings
2024-01-08 12:08:01,627 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Loading settings from /home/jo/code/kaggle/wandb/settings
2024-01-08 12:08:01,627 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2024-01-08 12:08:01,627 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-08 12:08:01,627 INFO    MainThread:21055 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-08 12:08:01,627 INFO    MainThread:21055 [wandb_init.py:_log_setup():524] Logging user logs to /home/jo/code/kaggle/wandb/run-20240108_120801-2yjqu2c7/logs/debug.log
2024-01-08 12:08:01,628 INFO    MainThread:21055 [wandb_init.py:_log_setup():525] Logging internal logs to /home/jo/code/kaggle/wandb/run-20240108_120801-2yjqu2c7/logs/debug-internal.log
2024-01-08 12:08:01,628 INFO    MainThread:21055 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f61503b37f0>
2024-01-08 12:08:01,628 INFO    MainThread:21055 [wandb_init.py:init():564] calling init triggers
2024-01-08 12:08:01,628 INFO    MainThread:21055 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.005, 'momentum': 0.9, 'epochs': 5, 'loss_function': 'tf.keras.losses.BinaryCrossentropy()', 'architecture': 'MLP'}
2024-01-08 12:08:01,628 INFO    MainThread:21055 [wandb_init.py:init():589] re-initializing run, found existing run on stack: v5e4pg4b
2024-01-08 12:08:01,630 INFO    MainThread:21055 [wandb_run.py:_finish():1962] finishing run jcrone/Titanic/v5e4pg4b
2024-01-08 12:08:01,630 INFO    MainThread:21055 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-08 12:08:01,630 INFO    MainThread:21055 [wandb_run.py:_restore():2185] restore
2024-01-08 12:08:01,630 INFO    MainThread:21055 [wandb_run.py:_restore():2191] restore done
2024-01-08 12:08:07,109 INFO    MainThread:21055 [wandb_run.py:_footer_history_summary_info():3837] rendering history
2024-01-08 12:08:07,110 INFO    MainThread:21055 [wandb_run.py:_footer_history_summary_info():3869] rendering summary
2024-01-08 12:08:07,113 INFO    MainThread:21055 [wandb_run.py:_footer_sync_info():3796] logging synced files
